tests/acceptance/test_organization_developer_settings.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+from sentry.testutils import AcceptanceTestCase
+
+
+class OrganizationIntegrationAcceptanceTest(AcceptanceTestCase):
```

Comment:
Could we rename this `OrganizationDeveloperSettingsAcceptanceTest` (or something similar), so that it's clearer what this test is for ?

Reply:
@MeredithAnya done